input/fsh/VS_Other.fsh

```diff
@@ -1,3 +1,29 @@
+ValueSet: MedDRAVS
+Id: meddra-value-set
+Title: "MedDRA Value Set"
+Description: "All codes from Medical Dictionary for Regulatory Activities Terminology (MedDRA), American English Equivalents, Version 7.0."
+* include codes from system MDR 
+
+ValueSet: CTCAdverseEventGradeVS
+Id: ctc-adverse-event-grade-value-set
+Title: "CTC Adverse Event Grade Value Set"
+Description: "Common terminology criteria grades associated with the severity of an adverse event. The additional term, 'free of symptoms' (aka grade 0) is used to indicate the patient has been assessed, and the given adverse event has not occurred."
+//* include codes from SCT where concept is-a #446401000124104 "National Cancer Institute common terminology criteria for adverse event grade finding (finding)"
+* SCT#162467007 "Free of symptoms (situation)"
```

Comment:
As you know, SCT#162467007 "Free of symptoms (situation)" would be invalid for CTCAE so if we use this for reporting, it would be rejected.